src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -117,14 +120,49 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
   }
 
   toPreviousPage() {
+    this.validationErrors = [];
     let caseEventData: CaseEventData = this.buildCaseEventData();
     this.updateFormData(caseEventData);
     this.previous();
     this.setFocusToTop();
   }
 
+  // Adding validation message to show it as Error Summary
+  generateErrorMessage(fields: CaseField[]) {
+    fields.filter(casefield => !this.caseFieldService.isReadOnly(casefield))
+          .filter(casefield => !this.pageValidationService.isHidden(casefield, this.editForm.getRawValue()))
+          .forEach(casefield => {
+            const fieldElement = this.editForm.controls['data'].get(casefield.id);
+
+            if (fieldElement && fieldElement.hasError('required')) {
```

Comment:
Instead of checking for `fieldElement` on each line of this, wrap it in a check for `fieldElement`:
```
if (fieldElement) {
  if (fieldElement.hasError('required')) {
    // stuff
  } else if (fieldElement.hasError('pattern')) {
    // more stuff
  } ...
}
```

Reply:
Did the changes as mentioned